### PR TITLE
Add integration with sentry

### PR DESCRIPTION
**Pull request by subdavis**
Looks like they have a new library to replace Raven with a simpler API surface.  :tada:

### CODE REVIEW

File: web/src/main.js [hunk 1]
```diff
@@ -15,6 +17,11 @@ Vue.config.productionTip = false;
 
 ApiService.init();
 
+SentryInit({
+  dsn: 'https://9c7281f0dfc5402a953698256bd067e0@sentry.io/1437129',
+  integrations: [new SentryVue({ Vue })],
+});
```
Comment (jbeezley) [inline web/src/main.js]: Might want to only do this in the production environment.

Comment (subdavis) [reply web/src/main.js]: I believe the library handles this for you.... let me check.

Comment (jbeezley) [reply web/src/main.js]: I got a notification, but maybe you were testing in production mode.

Comment (subdavis) [reply web/src/main.js]: Nope, you're right.  Need to check myself.

**Pull request merged**